src/components/BurnerWalletCard/NoBurner.tsx

```diff
@@ -67,3 +82,11 @@ export default function () {
     </div>
   )
 }
+
+export default function () {
+  return (
+    <Suspense fallback="">
```

Comment:
Still not sure, if need a fallback here

Reply:
We need it because of hook with store values